README.md

```diff
@@ -12,13 +12,23 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
 
 ## Quick Start
 
-1. Install `vagrant-service-manager` plugin
+1. Install `vagrant-service-manager` plugin:
 
         vagrant plugin install vagrant-service-manager
 
-2. Get the [Vagrantfile](Vagrantfile) and start the ADB using `vagrant up`. [More](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst) documentation on setting up ADB.
+2. Get the [Vagrantfile](Vagrantfile).
 
-3. Run the plugin to get environment variables and certificates
+3. Enable your desired provider(s) in [Vagrantfile](Vagrantfile) as:
+
+        config.servicemanager.providers = 'docker'
```

Comment:
we should show an example of starting multiple services

Also, even though these are container providers, perhaps we should call them services or use the word provision i.e.

config.servicemanager.services = 'docker'
or
config.servicemanager.provision = 'docker'